### PR TITLE
[github] Add ClangIR to new-prs-labeler.yml

### PR DESCRIPTION
**Pull request by lanza**

### CODE REVIEW

**Comment by boomanaiden154** [comment]
Make sure that the ClangIR team is created first before landing this so that everything works as expected.

**Comment by lanza** [comment]
> Make sure that the ClangIR team is created first before landing this so that everything works as expected.

What do you mean here by "ClangIR team is created"?

**Review by cor3ntin — COMMENTED** [review]
No needs for team to label PRs.
However, there needs to be a team for notifications to work.
Maybe Tom/Tobias can help with teams creation.

The name of the label/team must match and it needs to be all lowercase afaik.

**Comment by tstellar** [comment]
> No needs for team to label PRs. However, there needs to be a team for notifications to work. Maybe Tom/Tobias can help with teams creation.
> 
> The name of the label/team must match and it needs to be all lowercase afaik.

Uppercase labels are OK. I've added the team now.

File: .github/new-prs-labeler.yml [hunk 1]
```diff
@@ -1,3 +1,8 @@
+ClangIR:
+  - clang/include/clang/CIR/**/*
+  - clang/lib/CIR/**/*
+  - clang/tools/cir-*/**/*
```
Comment (bcardosolopes) [inline .github/new-prs-labeler.yml:4]: Should we add tests directories too?

**Review by bcardosolopes — APPROVED** [review]

**Pull request merged**